ha/core/system_health/status_mapper.py

```diff
@@ -65,12 +66,15 @@ def __init__(self):
     def map_event(self, event: HealthEvent) -> str:
         """Returns the status by mapping it against the source and event type."""
         try:
+            status = self.EVENT_TO_STATUS_MAPPING[event.event_type]
+            if event.specific_info and event.specific_info.get(SPECIFIC_INFO_ATTRIBUTES.FUNCTIONAL_TYPE.value) and \
```

Comment:
Will get(SPECIFIC_INFO_ATTRIBUTES.FUNCTIONAL_TYPE.value) cause exception for non node events ?
Please check